README.md

```diff
@@ -10,15 +10,13 @@ Currently the path to the compilers, ramdisks and kernels are hardcoded, but eas
 ## Usage:
 
 1. Clone directly into your AOSP tree (or make sure `ANDROID_BUILD_TOP` is set to the right path).
-2. Call `./oot/oot.sh <device name>`
-   Keep in mind that not all devices have a script yet.
+2. Call `./oot/oot.sh <device name>` (use `-h` to see more options)
 3. Flash the images. For simplicity, you can also add the `-f` option to the command, that will flash and `fastboot reboot` the device after building.
 
 ## TODO:
-- Add remaining devices
 - Move shared definitions to platforms? This includes sharing part of the cmdline across everything.
+- Expand platforms passed to this script
```

Comment:
Add period at the end of the sentence?

Reply:
Usually use `;` for this. Should be cleaned up now, most other points were anyway duplicates and/or already resolved.